### PR TITLE
adding missing components allowed for infrastructure nodes

### PR DESCRIPTION
**Pull request by ahardin-rh**
Continuing the work of https://github.com/openshift/openshift-docs/pull/26920

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@dmesser @vagnerfarias PTAL. Thank you!

**Comment by ahardin-rh** [comment]
@vagnerfarias I know you had some other content suggestions; I am reviewing your notes and will open a new PR with those suggestions as well. Thank you!

**Comment by ahardin-rh** [comment]
@dmesser @vagnerfarias PTAL

**Review by dmesser — APPROVED** [review]

File: modules/infrastructure-components.adoc [hunk 1]
```diff
@@ -6,13 +6,16 @@
 [id="infrastructure-components_{context}"]
 = {product-title} infrastructure components
 
-The following {product-title} components are infrastructure components:
+The following is the list of eligible infrastructure workloads that do not incur {product-title} worker subscriptions.
```
Comment (kalexand-rh) [inline modules/infrastructure-components.adoc:9]: Maybe:
```suggestion
The following infrastructure workloads do not incur {product-title} worker subscriptions:
```

**Review by kalexand-rh — COMMENTED** [review]
I have a suggestion, but this LGTM.

**Pull request merged**